### DISCUSSION TITLE
How about using Tauri instead of Electron?

### DISCUSSION THREAD

**Discussion by ImBIOS** [agalwood/Motrix#1487] [Ideas]
https://tauri.app/
https://www.electronjs.org/
https://blog.logrocket.com/tauri-electron-comparison-migration-guide/

**Comment by agalwood** [comment]
I don't consider migrating from Electron to Tauri at all, the cost of migration is large and the benefits are not obvious. Also, automatic updates may not be compatible with existing AutoUpdaters.

A brand new project might consider trying Tauri.

**Reply by ImBIOS** [reply]
Yeah, you're right. I've done some research on it. A huge amount of work required to migrate it just to get a little bit performance advantages.

**Comment by Taoister39** [comment]
I'm working on this, contributions are welcome.
https://github.com/Taoister39/tauri-motrix
